src/main/java/foodwhere/model/stall/Stall.java

```diff
@@ -72,6 +72,14 @@ public Set<Review> getReviews() {
         return Collections.unmodifiableSet(reviews);
     }
 
+    /**
```

Comment:
This makes the stall mutable, which makes it even more costly to implement `undo`. Immutable stalls are also better for defensive programming.